app/authentication/auth.py

```diff
@@ -44,11 +45,41 @@ def requires_no_auth():
     pass
 
 
-def restrict_ip_sms():
+def check_route_secret():
     # Check route of inbound sms (Experimental)
     # Temporary custom header for route security
-    if request.headers.get("X-Custom-forwarder"):
-        current_app.logger.info("X-Custom-forwarder received")
+    if request.headers.get("X-Custom-Forwarder"):
+        route_secret_key = request.headers.get("X-Custom-Forwarder")
+
+        # if route_secret_key is None:
```

Comment:
Removed commented out code

Reply:
We need this when we enable blocking once we check everything is OK.